src/pages/ManageComputeHost/DeleteComputeHost.js

```diff
@@ -0,0 +1,25 @@
+// (c) Copyright 2019 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import DeleteCompute from '../ReplaceServer/DeleteCompute';
+
```

Comment:
might need a bit of explanatory text here... its not obvious why this exists as a standalone child class

Reply:
I guess I was anticipating needing to customize this one a little more but didn't need to. So really all this gets you is the input props making more sense. oldServer vs target.